packages/frontend/core/src/components/page-list/view/edit-collection/hooks.tsx

```diff
@@ -25,6 +30,7 @@ export const useSelectPage = ({
         height="80%"
         overlayOptions={{ style: { backgroundColor: 'transparent' } }}
         contentOptions={{
+          'aria-describedby': undefined,
```

Comment:
Is this prop necessary for all `DialogContent`? If so, I think it's better to provide default value globally because we may forget to add this.

Reply:
![QQ_1721108596413](https://github.com/user-attachments/assets/b5e68845-9af5-4403-8bb8-098bc5352d58)
This property was added to resolve the warning in the image, and I have added it directly globally.